declarative.py

```diff
@@ -109,6 +109,6 @@ def be_fs_expanded(dev, path):
     elif fs == "btrfs":
         run(f"btrfs filesystem resize max {path}")
     elif fs == "xfs":
-        run(f"xfs_growfs -d {dev}")
+        run(f"xfs_growfs -d {path}")
```

Comment:
Please open another PR based on the develop branch,  and only include this change